app/validators/words_count_validator.rb

```diff
@@ -0,0 +1,11 @@
+class WordsCountValidator < ActiveModel::EachValidator
```

Comment:
I think rails has this out of the box i.e.

`validates_length_of : fee_details, :maximum => 250, :too_long => "Fee details can't be longetr than {{count}} words."), :tokenizer => lambda {|str| str.scan(/\w+/) }`

https://apidock.com/rails/ActiveRecord/Validations/ClassMethods/validates_length_of

Reply:
nice one

Reply:
oh dear
 Removed deprecated :tokenizer in the length validator 
https://github.com/rails/rails/commit/6a78e0ecd6122a6b1be9a95e6c4e21e10e429513